guardian/pressreader
# Add fake write to S3

## What does this change?

This change adds writing some dummy data to an S3 bucket in preparation for needing to do this with real data shortly.

## How to test

Checkout this project locally and follow the instructions to run the lambda, you should see output indicating a file is successfully written.

This branch has been deployed and data is successfully written to S3.

## How can we measure success?

Future work can be done more easily as the infra/code to write to S3 is in place.
